tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -446,6 +446,11 @@ internal static class FeatureFlags
             /// This flag defaults to true and is here in case customers need retrocompatibility only
             /// </summary>
             public const string HeaderTagsNormalizationFixEnabled = "DD_TRACE_HEADER_TAG_NORMALIZATION_FIX_ENABLED";
+
+            /// <summary>
+            /// Enables experimental support for activities
+            /// </summary>
+            public const string ActivitiesSupportEnabled = "DD_TRACE_ACTIVITIES_SUPPORT_ENABLED";
```

Comment:
Two thoughts: Is "support" redundant in the name? It's not used in any other setting name. And should we make the distinction between _listening_ to activities (this PR) and _emitting_ activities (possible future feature?)?

Combining the two, I thought we could use:
```csharp
ActivityListenerEnabled = "DD_TRACE_ACTIVITY_LISTENER_ENABLED";
```

Reply:
I agree, that's a better name, I made the change in https://github.com/DataDog/dd-trace-dotnet/pull/2446/commits/67df33aac005c8da654dfce5ca6d5ae1081375ea